cmd/postgres_exporter/pg_setting.go

```diff
@@ -30,6 +31,11 @@ var (
 	}
 )
 
+var (
+	// cache the first server setting when there are multiple servers
+	shortDescCache sync.Map
```

Comment:
I think instead of adding a sync.Map, I would be more inclined to use an empty or generic static string. I don't know that caching the first and always using that value provides more value. 

Do you disagree @SuperQ?

Reply:
Yes, I think these should be converted to static strings. In large deployments, having multiple versions of the help string bloats the `metadata` endpoints. We had similar problems with `jmx_exporter` where we had hundreds of different help strings, which made for huge (many megabyte) metadata json, which would slow down Grafana a lot.

Reply:
Thanks @sysadmind & @SuperQ 

I suggested a new solution. Please let me know if you expect a different approach